sdks/python/apache_beam/pvalue.py

```diff
@@ -223,20 +225,14 @@ def __init__(self, tag, value):
     self.value = value
 
 
-class PCollectionView(PValue):
-  """An immutable view of a PCollection that can be used as a side input."""
-
-  def __init__(self, pipeline, window_mapping_fn):
-    """Initializes a PCollectionView. Do not call directly."""
-    super(PCollectionView, self).__init__(pipeline)
-    self._window_mapping_fn = window_mapping_fn
+class AsSideInput(object):
```

Comment:
In comment mentions, this should not be used directly.

(Do you think it would be better to merge comments for AsXYZ here as a single source for common information?)